src/store/defaultSlice.js

```diff
@@ -66,41 +71,39 @@ const initialState = {
   theme: themeUtils.initializeTheme(),
 };
 
-const rootReducer = (state = initialState, action) => {
-  switch (action.type) {
-    case 'dashboard_update':
-      return { ...state, data: action.payload };
-    case 'is_version_allowed_update':
-      return { ...state, isVersionAllowed: action.payload.allowed };
-    case 'api_load_error_update':
-      return { ...state, apiLoadError: action.payload.apiLoadError };
-    case 'update_server_info':
-      return setServerInfo(state, action);
-    case 'toggle_theme':
-      return { ...state, theme: action.payload };
-    default:
-      return state;
-  }
-};
-
-/**
- * Set the server info a.k.a '/version' data on storage.
- * Will update keys based on default values.
- *
- * @param {ReduxStore} state - Current store state.
- * @param {payload} ServerInfo - Server info to save on storage.
- * @returns {ReduxStore} New state for the store.
- */
-const setServerInfo = (state, { payload }) => {
-  const serverInfo = cloneDeep(payload);
-  // Default values
-  serverInfo.decimal_places = serverInfo.decimal_places ?? constants.DECIMAL_PLACES;
-  serverInfo.native_token = serverInfo.native_token ?? constants.DEFAULT_NATIVE_TOKEN_CONFIG;
+const dashboardSlice = createSlice({
+  name: 'dashboard',
```

Comment:
I think I just didn't understand the name of the slice

Reply:
I didn't give it much thought before. Renamed to `rootSlice` for clarity on https://github.com/HathorNetwork/hathor-explorer/pull/384/commits/48a576f9c4c977efcf05b1a516184f768222f9f2